doc/main/reference/task_group_extensions.rst

```diff
@@ -62,43 +60,19 @@ Synopsis
 Member Functions
 ----------------
 
-.. cpp:function:: task_group(task_group_context& context)
-
-Constructs an empty ``task_group``, which tasks are associated with the ``context``.
-
-
 .. cpp:function:: template<typename F> task_handle  defer(F&& f)
 
-Creates a deferred task to compute ``f()`` and returns ``task_handle`` pointing to it.
-
-The task is not scheduled for execution until explicitly requested. For example, with the ``task_group::run`` method.
-However, the task is still added into the ``task_group``, thus the ``task_group::wait`` method waits until the ``task_handle`` is either scheduled or destroyed.
-
-The ``F`` type must meet the `Function Objects` requirements described in the [function.objects] section of the ISO C++ Standard.
-
 As an optimization hint, ``F`` might return a ``task_handle``, which task object can be executed next.
 
 .. note::
    The ``task_handle`` returned by the function must be created with ``*this`` ``task_group``. It means, with the one for which run method is called, otherwise it is an undefined behavior. 
```

Comment:
```suggestion
   The ``task_handle`` returned by the function must be created using ``*this`` ``task_group``. I.e. one for which the run method is called, otherwise it is an undefined behavior. 
```

Reply:
same here